test/build_function_tests/stencil-broadcast-outplace.jl

```diff
@@ -1,16 +1,27 @@
 :(function (x,)
-      let ˍ₋out = zeros(Float64, map(length, (1:6, 1:6)))
-          begin
-              ˍ₋out_2_input_1 = (broadcast)(+, x, (adjoint)(x))
-              ˍ₋out_1 = (view)(ˍ₋out, 1:6, 1:6)
-              ˍ₋out_1 .= 0
-              ˍ₋out_2 = (view)(ˍ₋out, 2:5, 2:5)
-              for (j, j′) = zip(Base.OneTo(4), reset_to_one(Base.OneTo(4)))
-                  for (i, i′) = zip(Base.OneTo(4), reset_to_one(Base.OneTo(4)))
-                      ˍ₋out_2[i′, j′] = (+)(ˍ₋out_2[i′, j′], (+)(1, (getindex)(ˍ₋out_2_input_1, i, j)))
-                  end
+      let _out = (zeros)(Float64, (map)(length, (1:6, 1:6))), var"##334" = begin
+                  _out_2_input_1 = (broadcast)(+, x, (adjoint)(x))
+                  _out_1 = (view)(_out, 1:6, 1:6)
+                  var"##335" = (Symbolics.broadcast_assign!)(_out_1, 0)
+                  _out_2 = (view)(_out, 2:5, 2:5)
+                  var"##336" = for var"##338" = (zip)(Base.OneTo(4), (Symbolics.reset_to_one)(Base.OneTo(4)))
+                          begin
+                              j = var"##338"[1]
+                              j′ = var"##338"[2]
+                              for var"##337" = (zip)(Base.OneTo(4), (Symbolics.reset_to_one)(Base.OneTo(4)))
```

Comment:
```suggestion
                  var"##336" = for var"##338" = (zip)(1:4, (Symbolics.reset_to_one)(1:4))
                          begin
                              j = var"##338"[1]
                              j′ = var"##338"[2]
                              for var"##337" = (zip)(1:4, (Symbolics.reset_to_one)(1:4))
```

Reply:
Why? It's equivalent.

Reply:
The test on this failed. Maybe a change somewhere accidently changed its behavior

Reply:
Oh, interesting.

Reply:
What do you think about `reset_to_one`? I only included it in the expression because the old version did, but I don't see a reason why we can't just evaluate and insert the value of `reset_to_one(..)`

Reply:
I tested this Symbolics branch locally with https://github.com/AayushSabharwal/SymbolicUtils.jl/tree/as/better-cse